tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "compilerOptions": {
-    "outDir": "./build/",
+    "outDir": "./dist/",
     "sourceMap": true,
     "noImplicitAny": true,
     "module": "ES6",
     "target": "ES5",
```

Comment:
Do we want to keep that ES5 in the future? When we drop IE support we might want a more modern bundle but at the same time some of Sumo clients might need to keep using ES5. We might need multiple TS outputs. Something to think about in the future.